test/Microsoft.DotNet.ProjectModel.Tests/LibraryExporterPackageTests.cs

```diff
@@ -336,7 +336,10 @@ private LibraryExport ExportSingle(LibraryDescription description = null)
             var rootProject = new Project()
             {
                 Name = "RootProject",
-                CompilerName = "csc"
+                _defaultCompilerOptions = new CommonCompilerOptions
```

Comment:
Yes. This will be changed when that is reviewed. https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.ProjectModel/Project.cs#L19